src/lib/y2network/backend.rb

```diff
@@ -87,5 +89,9 @@ def self.by_id(id)
     def available?
       Yast::NetworkService.is_backend_available(id)
     end
+
+    def packages
+      self.class.const_get(:PACKAGES)
```

Comment:
np: to be honest, it looks a little bit over-engineered. Just defining a method that returns the list should be enough. Additionally, if I wanted to create a new derived class, I would need to know that I need to define a constant inside the class to define the list of packages (it is a kind of indirection).

Anyway, that's just an opinion, feel free to keep it as it is because I know you already have tested the code and I do not want to invalidate those tests :-)

Reply:
> np: to be honest, it looks a little bit over-engineered. Just defining a method that returns the list should be enough. Additionally, if I wanted to create a new derived class, I would need to know that I need to define a constant inside the class to define the list of packages (it is a kind of indirection).

Just keep it simple.

> 
> Anyway, that's just an opinion, feel free to keep it as it is because I know you already have tested the code and I do not want to invalidate those tests :-)